lavender.json

```diff
@@ -0,0 +1,29 @@
+{
+   "response":[
+      {
+         "maintainer":"Prashant",
+         "oem":"Xiaomi",
+         "device":"Redmi Note 7",
+         "filename":"crDroidAndroid-12.1-20220702-lavender-GMS-v8.6.zip",
+         "download":"https://polished-disk-a3d6.shubhamxd.workers.dev/0:/crDroidAndroid-12.1-20220702-lavender-GMS-v8.6.zip",
+         "timestamp":1656741464,
+         "md5":"1c59b4dec585bcc2b31803e90afdcf7c",
+         "sha256":"6b2ecf8867b96466a78725f869e9443b0fec97dfdc8f42a16e890e5fc9581956",
+         "size":1370311682,
+         "version":"8.6",
+         "buildtype":"Monthly",
+         "forum":"https://forum.xda-developers.com/t/rom-12-1-unofficial-lavender-crdroid-v8-6-25-06-2022.4464597/",
+         "gapps":"",
+         "firmware":"",
+         "modem":"",
+         "bootloader":"",
+         "recovery":"https://orangefox.download/device/lavender",
+         "paypal":"",
+         "telegram":"https://t.me/+yXeknMeipy05YTVl",
+         "crDroid-org": "https://github.com/crdroidandroid",
+         "dt":"https://github.com/Prashant-1695/device_xiaomi_lavender",
```

Comment:
modify this url to point to our repo hosted at crDroid GitHub
there are cloned repos already so check out